src/app/components/mnoe-api/marketplace.svc.coffee

```diff
@@ -20,7 +20,7 @@ angular.module 'mnoEnterpriseAngular'
 
     @addAppReview = (appId, data) ->
       payload = {app_review: data}
-      MnoeApiSvc.one('marketplace', parseInt(appId)).post('app_review', payload).then(
+      MnoeApiSvc.one('marketplace', parseInt(appId)).doPOST(payload, '/app_review').then(
```

Comment:
when doing post we are not able to get the whole app object in the response, but a simple review, regardless all the magic we do on the back end, it is due the mnoApiSvc too, cause the, and it is also the way is done in other parts of the project.

api.svc.coffee
```
          if (_.startsWith(what, '/'))
            extractedData = data
          # On getList extract and restangularize the objects list
          else if (operation == 'getList')
            extractedData = data[what]
          # Extract and restangularize the object
          else if (operation == 'get' || operation == 'post' || operation == 'put')
            what = inflector.singularize(what)
            extractedData = data[what]
          else
            extractedData = data
          return extractedData
      )
```

Due the fact we are getting the whole object instead of the single review with the regular post, the object in angular will be undefined.
            extractedData = data[what]